src/CLI/renderer/templates/show_vrf.j2

```diff
@@ -0,0 +1,13 @@
+{{'----------------------------------------------------------------'}}
+{{'VRF-NAME'.ljust(20)}}{{'INTERFACES'}}
+{% if json_output and 'openconfig-network-instance' in json_output -%}
+{% set inst_info = json_output['openconfig-network-instance'] %}
+{% set state = inst_info['state'] %}
```

Comment:
Which command format are we following here?

OS10:


Mgmt VRF:
---------------
S6K-12(config)# ip vrf management
S6K-12(conf-vrf)# interface management
S6K-12(conf-vrf)#

Data VRF:
-------------
S6K-12(config)# ip vrf blue
S6K-12(conf-vrf)#
S6K-12(config)# interface vlan 10
S6K-12(conf-if-vl-10)# ip vrf forwarding blue
S6K-12(conf-if-vl-10)#


CISCO:

ct-ms-cisco-01(config)# vrf context management
ct-ms-cisco-01(config-vrf)#
ct-ms-cisco-01(config)# int mgmt 0
ct-ms-cisco-01(config-if)# vrf member management
ct-ms-cisco-01(config-if)#

ct-ms-cisco-01# show vrf
VRF-Name                           VRF-ID State   Reason
default                                 1            Up      --
management                      2            Up      --
ct-ms-cisco-01#

Arista:

ct-ms-arista(config)#vrf definition MGMT
ct-ms-arista(config)#int management 1
ct-ms-arista(config-if-Ma1)#vrf forwarding MGMT
ct-ms-arista(config-if-Ma1)#

ct-ms-arista#show vrf
   Vrf              RD              Protocols       State         Interfaces
---------------- --------------- --------------- ---------------- ----------
   MGMT      <not set>                       incomplete
   blue             2:2             ipv4            no routing    Ethernet47
   management       <not set>                       incomplete
   red              1:1             ipv4            no routing    Ethernet46
   test             <not set>                       incomplete

ct-ms-arista#